word-guess-classes.rb

```diff
@@ -0,0 +1,235 @@
+require 'colorize'
+
+class Word
+  attr_accessor :guessed_letters, :blanks, :wrong_guesses, :letters, :name
+
+  def initialize(word)
+    @name = word
+    @letters = word.chars
+    @blanks = []
+    @guessed_letters = []
+    @guess = ""
+    @wrong_guesses = 0
+    create_blank_array
+  end
+
+  def create_blank_array
+    @letters.length.times do
+      @blanks << "_"
+    end
+  end
+
+  def print_blanks
+    @blanks.each do | value |
+      print value
+    end
+  end
+
+  def check_guess(guess)
+    if @guessed_letters.include?(guess)
+      puts "\nYou've already guessed this letter.".colorize(:yellow)
+    else
+      @guessed_letters << guess
+    end
+    if !@letters.include?(guess)
+      if guess.empty?
+        puts "No letter inputted. Try again."
+      else
+        @wrong_guesses += 1
+        puts "\nYou guessed wrong! Try again.".colorize(:red)
+      end
+    else
+        puts "\nCorrect!!!".colorize(:green)
+    end
+  end
+
+  def fill_blanks(guess)
+    @letters.each.with_index do |letter, index|
+      if letter == guess
+        @blanks[index] = guess
+      end
+    end
+  end
+
+  def print_guesses
+    print "\nGuessed letters: "
+    @guessed_letters.each do | value |
+      print value.upcase
+      print " "
+    end
+  end
+
+end
+
+class Image
+  def initialize
+    @image0 = [
+      "                           __/ \\--\\".colorize(:red),
```

Comment:
Since you stick all these images into an array, they probably don't also need to be instance variables. Local variables would be sufficient.